jscomp/Makefile

```diff
@@ -541,10 +541,12 @@ bin/bspp.ml:./bin/bspack.exe
 bin/js_compiler.ml:./bin/bspack.exe
 	 ./bin/bspack.exe -D BS_COMPILER_IN_BROWSER=true -U BS_DEBUG -bs-MD  -prelude-str 'module Config = Config_whole_compiler' -bs-exclude-I config -o $@ -bs-main Jsoo_main -I $(OCAML_SRC_UTILS) -I $(OCAML_SRC_PARSING) -I $(OCAML_SRC_TYPING) -I $(OCAML_SRC_BYTECOMP) -I $(OCAML_SRC_DRIVER) -I stubs -I ext -I syntax -I depends -I common -I core
 
-bin/jsoo_reactjs_jsx_ppx_v2.ml:./bin/reactjs_jsx_ppx_v2.bspp.ml
-	make -C bin bspp.exe
+bin/jsoo_reactjs_jsx_ppx_v2.ml:./bin/reactjs_jsx_ppx_v2.bspp.ml ./bin/bspp.exe
 	BS_COMPILER_IN_BROWSER=true ./bin/bspp.exe $^ > $@
 
+bin/reactjs_jsx_ppx_v2.ml:./bin/reactjs_jsx_ppx_v2.bspp.ml ./bin/bspp.exe
```

Comment:
We now generate `reactjs_jsx_ppx_v2.ml` from `reactjs_jsx_ppx_v2.bspp.ml`

Reply:
Nice!